observability/readme.md

```diff
@@ -9,9 +9,64 @@ Building observable systems enables development teams at CSE to measure how well
 3. Measure operational performance of the system.
 4. Identify and diagnose failures to get to the problem fast.
 
+## Pillars of observability
+
+### Logging
+
+Discrete events with the goal of helping engineers identify problem area(s) during failures.
+
+- Pay attention to logging levels. Logging too much will increase costs and decrease application throughtput.
+- Ensure logging configuration can be modified without code changes. Ideally, make it changeable without application restarts.
+- If available, take advantage of logging levels per category allowing granular logging configuration.
+- Check for log levels before logging, thus avoiding allocations and string manipulation costs.
+- Ensure service versions are included in logs, thus having the capability of identifying problematic releases.
```

Comment:
Simpler structure - "Ensure service versions are included in logs to be able to identify problematic releases."